components/command_palette/utils.ts

```diff
@@ -88,6 +88,20 @@ export function channelToCommandPaletteItemTransformer(channels: Channel[], team
     });
 }
 
+export function defaultsTransformer(items: any[], teams: Record<string, Team>): CommandPaletteItem[] {
+    return items.filter((item) => Boolean(item)).map((item) => {
+        return {
+            teamName: teams[item.teamId]?.name,
+            pictograph: {
```

Comment:
I've put this in for now until I figure out how to get a custom icon across the interface.

Reply:
@lieut-data Do we have custom icons for Playbooks as well? In design @abhijit-singh has used single icon for Playbooks